packages/lit-element/src/demo/ts-element.ts

```diff
@@ -1,22 +1,24 @@
-import {html, LitElement} from '../lit-element.js';
-import {property} from '../lib/decorators.js';
+import { html, LitElement } from "../lib/lit-element.js";
```

Comment:
Use this Prettier config: https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-html/.prettierrc.json
Ok to copy for now I guess, but we should hoist it to the top level soon.

Reply:
Done.